pkg/cluster/nodes/node.go

```diff
@@ -398,13 +427,29 @@ func NeedProxy() bool {
 // See also: NeedProxy and getProxyDetails
 func (n *Node) SetProxy() error {
 	details := getProxyDetails()
+	// don't proxy localhost and docker network
+	ip, err := n.IP()
```

Comment:
Why should we need to do the IP and the prefix? 🤔

Reply:
I thin kthat we need to obtain the docker network CIDR so we can add it to the NO_PROXY var , and because we are into a node context the only info that we have is the current node ip and the prefix len, this way we can build the docker network as `IPAddress/IPPrefixLen`

```
        "NetworkSettings": {
            "Bridge": "",
(snip)
            "Gateway": "172.17.0.1",
            "GlobalIPv6Address": "2001:db8:1::242:ac11:4",
            "GlobalIPv6PrefixLen": 64,
            "IPAddress": "172.17.0.4",
            "IPPrefixLen": 16,
(snip)
```

From a networking point of view, it´s perfectly valid as CIDR, don´t know if this can cause any issue with docker

Reply:
Ok I see that now, can we explicitly build the CIDR in a separate function and just consume than here?